core/doc_data.cpp

```diff
@@ -152,9 +152,10 @@ void DocData::method_doc_from_methodinfo(DocData::MethodDoc &p_method, const Met
 
 	return_doc_from_retinfo(p_method, p_methodinfo.return_val);
 
-	for (int i = 0; i < p_methodinfo.arguments.size(); i++) {
+	int i = 0;
+	for (List<PropertyInfo>::ConstIterator itr = p_methodinfo.arguments.begin(); itr != p_methodinfo.arguments.end(); ++itr, ++i) {
```

Comment:
Putting both `itr` and `i` in the `for` to ensure they iterate in unison

Reply:
Instead `for`, how about `std::for_each` with lambda expression for readability?
idk it improve performance.

Reply:
We avoid using the STL, so not applicable here